backend/controllers/Auth.js

```diff
@@ -0,0 +1,29 @@
+const router = require("express").Router();
+const User = require("../models/User");
+
+router.post("/register", async (req, res) => {
+  try {
+    const { firstName, lastName, email } = req.body;
+    if (!firstName || !lastName || !email) {
+      throw new Error("The user schema is not complete");
+    }
```

Comment:
To add in email validation we can do 

``` 
const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
    if (!emailRegex.test(email)) {
      throw new Error("Invalid email address");
    }
```
a regular expression (emailRegex) is used to validate the email format. The regular expression pattern /^[^\s@]+@[^\s@]+\.[^\s@]+$/ checks if the email address has the basic structure of <username>@<domain>.<tld>